manifests/pvc.yaml

```diff
@@ -0,0 +1,14 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: jenkins-local
+spec:
+  accessModes:
+    - ReadWriteOnce
+    - ReadWriteMany
```

Comment:
do we need `ReadWriteMany` ?

Reply:
Not strictly, but I like to include it so I can just start up e.g. a cosa pod and poke around the PVC.